week-2-practise/level-3/5-indexOf.js

```diff
@@ -5,8 +5,9 @@
 // - NOT change the original array
 // - return a new array with the replacement value inserted
 
-function replace(arr, valueToReplace, newValue) {
-  return; // complete this statement
+function replace(numbers, valueToReplace, newValue) {
+  numbers.splice(numbers.indexOf(valueToReplace), 1, newValue);
+  return numbers; // complete this statement
```

Comment:
You need to create a new array to avoid changing the original array. Once you fix that problem see if the output matches the expected output